platform/core/src/services/HangingProtocolService/HangingProtocolService.js

```diff
@@ -357,6 +390,7 @@ class HangingProtocolService {
           SeriesInstanceUID: aSeries.SeriesInstanceUID,
           matchingScore: totalMatchScore,
           matchDetails: matchDetails,
+          displaySet: aSeries,
```

Comment:
I'm not sure if `displaySet` makes sense here, since this matching function should only care about metadata matching and sorting them. Why do we need `displaySet` here?